README.md

```diff
@@ -73,10 +73,10 @@ An object is identified as a built-in using:
 
 #### `[[Builtin]]` internal slot
 
-All built-ins, with the exception of the intrinsic object `%ObjectPrototype%`,
-would have a `[[Builtin]]` internal slot with a string value identifying the
-name of the built-in. For instance, for the `%Math%` intrinsic object, the
-value of `[[Builtin]]` is `'Math'`.
+All built-ins, with the exception of the intrinsic objects `%ObjectPrototype%`,
+and `%Object%` have a `[[Builtin]]` internal slot with a string value
```

Comment:
it's largely unnecessary given that `typeof obj === 'object'` and the fallback behavior for `Builtin.typeof` is to do `typeof obj`.